changhyun/app.js

```diff
@@ -0,0 +1,53 @@
+// 사용하는 npm 패키지(모듈)들 : express, nodemon, cors, dotenv, morgan
+
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const morgan = require('morgan'); //morgan 모듈 추가하기
+const dotenv = require('dotenv');
+
+const { DataSource } = require('typeorm'); //DataSource 모듈을 불러오기 
+
+dotenv.config(); //dotenv 사용
+// ES 모듈에서 환경 변수 불러오기(import) 다시 공부하기.
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+}) //process객체 : node.js 기본 글로벌 객체로 별도의 require 호출 없이 모든 모듈에서 접근이 가능 (process의 정확한 기능은 무엇인지?)
+
+myDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  })
+
+app.use(cors());
+app.use(morgan('combined')); //morgan 사용하기 (combined, common, dev, short, tiny)
+
+app.get('/ping', function (req, res, next){
+  res.json({message:'pong'});
+})
+app.listen(3000, () => { console.log('server listening on port 3000')});
```

Comment:
```suggestion
app.listen(PORT, () => { console.log(`server listening on port ${PORT}`)});
```
PORT는 하드코딩 하는 것이 아닌 .env에 환경변수로서 넣어둔 port 넘버를 import 해서 사용할 수 있게 해주세요!

이부분에서 이해가 안되는 부분이 있다면, 라운지에 있는 멘토 혹은 동기들한테 여쭤봐주세요 :)